pkg/app/piped/livestatestore/cloudrun/cloudrun.go

```diff
@@ -0,0 +1,79 @@
+// Copyright 2022 The PipeCD Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package cloudrun
+
+import (
+	"context"
+	"time"
+
+	"go.uber.org/zap"
+
+	provider "github.com/pipe-cd/pipecd/pkg/app/piped/cloudprovider/cloudrun"
+	"github.com/pipe-cd/pipecd/pkg/config"
+)
+
+type Store struct {
+	store    *store
+	logger   *zap.Logger
+	interval time.Duration
+}
+
+type Getter interface {
+	GetAppLiveServiceManifest(appID string) provider.ServiceManifest
```

Comment:
How about just naming it as `GetServiceManifest` since this function is already inside the package named as `livestatestore`?